bin/build_gfedb.py

```diff
@@ -90,7 +95,9 @@ def hla_alignments(dbversion):
         sth_prot = data_dir + "/../data/" + dbversion + "/" + loc.split("-")[1] + "_prot.sth"
 
         logging.info("Loading " + sth_gen)
-        align_gen = AlignIO.read(open(sth_gen), "stockholm")
+        print(skip_alleles)
+        #align_gen = AlignIO.read(open(sth_gen), "stockholm")
+        align_gen = next(AlignIO.parse(sth_gen, "stockholm"))
```

Comment:
Why the change to `parse` ?

Reply:
This has been moved back to AlignIO.read.  This was the wrong direction in resolving this issue.